.github/workflows/bootstrap.yml

```diff
@@ -0,0 +1,74 @@
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+
+name: Bootstrap
+on:
+  - pull_request # without merge conflicts
+  - push # branch or tag
```

Comment:
Maybe restrict this to just PRs / pushes to the `stable` branch?

Reply:
Sounds good to me. :+1:

Reply:
pull_request: stable
push: [ master, stable ]